<?xml version="1.0" encoding="UTF-8"?>
<testrun name="SolrTests" project="SolrTestApp1" tests="3" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="SolrTests" time="0.291">
    <testcase name="test_basic" classname="SolrTests" time="0.0"/>
    <testcase name="test_GetDocFromSolrService_UnitTest" classname="SolrTests" time="0.251"/>
    <testcase name="test_GetDocFromSolrService_AcctanceTest" classname="SolrTests" time="0.04">
      <failure>
        <expected>ma_34_447</expected>
        <actual>ma_1_1</actual>org.junit.ComparisonFailure: expected:&lt;ma_[34_447]&gt; but was:&lt;ma_[1_1]&gt;&#13;
	at org.junit.Assert.assertEquals(Assert.java:125)&#13;
	at org.junit.Assert.assertEquals(Assert.java:147)&#13;
	at SolrTests.test_GetDocFromSolrService_AcctanceTest(SolrTests.java:25)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:168)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:134)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:110)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:128)&#13;
	at junit.framework.TestResult.run(TestResult.java:113)&#13;
	at junit.framework.TestCase.run(TestCase.java:124)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:243)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:238)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:83)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:467)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:683)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:390)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:197)&#13;
&#13;
</failure>
    </testcase>
  </testsuite>
</testrun>
